### PR TITLE
bugfix: Show bugs which are not In Process column (Trello)

### PR DESCRIPTION
**Pull request by iaroslavfleancu**

### CODE REVIEW

**Comment by iaroslavfleancu** [comment]
[Show bugs which are not In Process column](https://trello.com/c/3xMLnGtH/1-show-bugs-which-are-not-in-process-column)

**Review by zolotokrylin — COMMENTED** [review]
@iaroslavfleancu it is very hard to do the code review of your commit, because you have done code formatting and it looks like you have introduced lots of changes in the code. But I believe this is not a case and this is just a code formatting.

In the future, if you (or your IDE) does code styling, make sure you do not push code styling as a separate commit. 

Right now, I want to do review of your functional changes introductions. What do you suggest I should do?

**Comment by zolotokrylin** [comment]
Also pull request name is not descriptive. What Test implementation means?

File: index.html [hunk 1]
```diff
@@ -212,10 +212,62 @@
 	
 	</script>
 
-	<div id="bugsNotInProcess">
-		<h1>Fix it!</h1>
-		<p>If you know there is a bug, you must fix it. <a href="https://www.ministryoftesting.com/dojo/lessons/ten-reasons-why-you-fix-bugs-as-soon-as-you-find-them">(Why?)</a></p>
-	</div>
+	<style>
```
Comment (zolotokrylin) [inline index.html]: @iaroslavfleancu move this into separate .css file

**Review by zolotokrylin — APPROVED** [review]

**Pull request merged**